src/VerifyWebauthnController.js

```diff
@@ -197,23 +197,14 @@ export default FormController.extend({
       return children;
     },
 
-    postRender: function () {
-      _.defer(() => {
-        if (webauthn.isNewApiAvailable()) {
-          this.model.save();
-        } else {
-          this.$('[data-se="webauthn-waiting"]').hide();
-        }
-      });
-    },
-
     _startEnrollment: function () {
       this.$('.okta-waiting-spinner').show();
       this.$('.o-form-button-bar').hide();
     },
 
     _stopEnrollment: function () {
       this.$('.okta-waiting-spinner').hide();
+      this.$('.o-form-button-bar [type="submit"]')[0].value = loc('retry', 'login');
```

Comment:
should be key `verify.u2f.retry` instead... I think that is what we had before, in case someone has customized it